dictionary.txt

```diff
@@ -1005,7 +1005,7 @@ imho
 Imoh
 impactful
 implementers
-implementors-of-Gatsby
+implements of Gatsby
```

Comment:
note: in this case is the best to remove this line and try if you it pass the linting - and if not then only add implements